packages/grid/_modules_/grid/models/gridExport.ts

```diff
@@ -1,8 +1,34 @@
+/**
+ * Available CSV delimiter characters used to separate fields.
+ * @default ','
+ */
+export type CsvDelimiterCharacter = ',' | ';';
+
 /**
  * The options to apply on the CSV export.
  */
 export interface GridExportCsvOptions {
+  /**
+   * If `true`, the comma character used to separate the integer part from the fractional part of a number.
+   * This flag also changes the delimiter from `,` to `;`.
+   * @default false
+   */
+  commaAsDecimalSeparator?: boolean;
```

Comment:
Could we remove this option and move it to a standalone PR? It seems to create friction for another change this we could merge as-is.

Reply:
Ok. I'll get rid off this feature in this PR.